addons/tmf_loadouts/loadouts/uk_rm_1982.hpp

```diff
@@ -0,0 +1,512 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
+//Dependencies:
+//		* CUP Units
+//		* CUP Weapons
+//		* Community Factions Project
+//		* FIR AWS(AirWeaponSystem)
+//		* MNP Uniforms
+//		* HLC FAL Pack
+//		* HLC MP5 Pack
+//		* ACE
+
+//		Note: Just swap the berets and you have a para regiment instead.
+//		Addtl. Note: Helmets are the only armor on this kit.
+
+//	Created by: AChesheireCat
+
+class baseMan {// Weaponless baseclass
+	displayName = "Unarmed";
+	// All randomized.
+	uniform[] = {
+		LIST_2("CUP_U_B_BDUv2_dirty_DPM"),
+		LIST_2("CUP_U_B_BDUv2_DPM"),
+		"CUP_U_B_BDUv2_dirty_DPM_OD",
+		"CUP_U_B_BDUv2_DPM_OD"
+	};
+	vest[] = {"CFP_Alice_Vest2"};
+	backpack[] = {
+		"CUP_B_AlicePack_Bedroll",
+		"CUP_B_AlicePack_Khaki"
+	};
+	headgear[] = {
+		"CFP_IDF_M76",
+		"SP_M1Helmet_GrayDim",
+		"CFP_IDF_M76_BD_Camo",
+		"CUP_H_BAF_MTP_Mk7_PRR_SCRIM_A",
+		"CUP_H_BAF_DPM_Mk6_NETTING_PRR"
+	};
+	goggles[] = {"default"};
+	hmd[] = {};
+	// Leave empty to remove all. "Default" > leave original item.
+	faces[] = {};
+	// Leave empty to not change faces.
+	insignias[] = {};
+	// Leave empty to not change insignias
+
+	// All randomized
+	primaryWeapon[] = {};
+	scope[] = {};
+	bipod[] = {};
+	attachment[] = {};
+	silencer[] = {};
+	// Leave empty to remove all. "Default" for primaryWeapon > leave original weapon.
+
+	// Only *Weapons[] arrays are randomized
+	secondaryWeapon[] = {};
+	secondaryAttachments[] = {};
+	sidearmWeapon[] = {};
+	sidearmAttachments[] = {};
+	// Leave empty to remove all. "Default" for secondaryWeapon or sidearmWeapon > leave original weapon.
+
+	// These are added to the uniform or vest
+	magazines[] = {};
+	items[] = {
+		 MEDICAL_R ,
+		"ACE_Canteen",
+		"ACE_EntrenchingTool",
+		"ACE_Flashlight_MX991"
+	};
+	// These are added directly into their respective slots
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch"
+	};
+
+	// These are put into the backpack
+	backpackItems[] = {"MNP_Beret_RM"};
+
+	// Unit traits, see https://community.bistudio.com/wiki/setUnitTrait
+	traits[] = {};
+
+	// This is executed after unit init is complete. argument: _this = _unit.
+	code = "";
+};
+
+class r : baseMan {
+	displayName = "Rifleman";
+	primaryWeapon[] = {"hlc_rifle_L1A1SLR"};
+	magazines[] = {
+		LIST_7("CUP_20Rnd_762x51_FNFAL_M"),
+		LIST_4("CUP_HandGrenade_L109A1_HE"),
```

Comment:
Lots of grenades there

Reply:
Yup - it's to make up for the lack of overall ammunition. A typical combat load with 5.56 is 8x 30 rnd mags while this is 7x 20 rnd mags. A LOT less ammunition.